.changeset/shy-carrots-love.md

```diff
@@ -0,0 +1,5 @@
+---
+"changelog": minor
+---
+
+Use the Rowan model for the CST i.e. TokenNodes containe the string content, and RuleNodes contain only the combined _length_ of their children's text.
```

Comment:
Looks like CI is failing because of misspelled `containe` here.

Reply:
done